src/core/utils/lang.ts

```diff
@@ -7,11 +7,13 @@ export interface Jsonable {
 
 /**
  * Returns true if the given value is a promise.
+ * (`instanceof Promise` doesn't work for angular promises)
  *
  * @param value Value to check
  */
-export function isPromise(value: any): value is Promise<any> {
-    return value instanceof Promise;
+export function isPromiseLike(value: any): value is Promise<any> | angular.IPromise<any> {
```

Comment:
Was it noted in changelog about this backwards incompatibility?

Reply:
Added